content/blog/automating-operations-on-dscc-using-ansible-playbooks.md

```diff
@@ -0,0 +1,440 @@
+
+Ansible helps the users/admins automate the deployment of resources and applications without the manual overhead of creating everything from scratch. These playbooks can be configured with conditions, variables, and tasks. Currently, simple playbooks, like performing CRUD operations on the resources, are available. These playbooks can be considered basic building blocks and can be reused to build simple-to-complex use cases.
+
+**[Ansible Modules for  Data Services Cloud Console](https://github.com/HewlettPackard/greenlake-data-services-ansible):**
+
+The following Ansible modules are currently available for Data Services Cloud Console. You can use the samples given or customize it - 
+
+*greenlake_audit_events_facts* - Get details of audit events
+
+*greenlake_host* - Manage host
+
+*greenlake_host_facts -* Get details of a host
+
+*greenlake_host_group -* Manage host group
+
+*greenlake_host_group_facts -* Get details of a host group
+
+*greenlake_host_initiator_facts -* Get details of a host initiator
+
+*greenlake_storage_system_facts -* Get details of a storage system
+
+*greenlake_volume -* Manage volume
+
+*greenlake_volume_facts -* Get details of a volume
+
+*greenlake_volumeset -* Manage volume set
+
+*greenlake_volumeset_facts -* Get details of a volume set
+
+**Prerequisites to use these Ansible playbooks:**
+
+1. Any machine with Python 3.8 or newer installed. This includes Red Hat, Debian, CentOS, macOS, any of the BSDs, and so on. (Microsoft Windows is not supported *)
+2. The latest and most stable version of Ansible must be installed. (current version is 2.9)
+3. HPE GreenLake Data Services Python SDK. (Installation procedure is mentioned below)
+4. Cloning the GitHub repo that has these playbooks
+5. Setup ANSIBLE_LIBRARY and ANSIBLE_MODULE_UTILS environment variables
+
+   (* If you are using Windows 10, then an Ubuntu terminal called Windows Subsystem for Linux(WSL) can be used.) 
+
+**Installing Ansible on Ubuntu**
+
+1. Make sure your system’s package index is up to date. Refresh the package index with the following command:
+
+   ```shell
+   $sudo apt update
+   ```
+2. Next, install Ansible on Ubuntu with the command:
+
+   ```shell
+   $sudo apt install ansible
+   ```
+
+The installation will prompt you to press Y to confirm, with the rest of the installation process being automated.
+
+3. Check the version of Ansible by using the following command:
+
+   ```shell
+   $ansible --version
+   ```
+
+For other operating systems, please refer to the official Ansible documentation.
+
+**Installing the HPE GreenLake Data Services Python SDK**
+
+These Ansible playbooks use the Python libraries of the HPE GreenLake Python SDK. Install the SDK using this command:
+
+**...Below command will change and use the newly available repository...**
+
+```shell
+$pip install git@github.hpe.com:sijeesh-kattumunda/greenlake_data_services_python.git
+```
+
+Cloning the DSCC GitHub repo	
```

Comment:
Cloning the Data Services Cloud Console GitHub repo

Also, I think the "Cloning the Data Services Cloud Console GitHub repo" should be highlighted in bold like the "Installing the HPE GreenLake Data Services Python SDK". Isn't it?